media/static-bundles.json

```diff
@@ -1080,13 +1086,19 @@
       ],
       "name": "firefox_whatsnew_93_cycle"
     },
+    {
+      "files": [
+        "js/firefox/whatsnew/up-to-date.js"
+      ],
+      "name": "firefox_whatsnew_95_de"
+    },
     {
       "files": [
         "js/firefox/whatsnew/up-to-date.js",
         "js/base/mozilla-fxa-product-button.js",
         "js/base/mozilla-fxa-product-button-init.js"
       ],
-      "name": "firefox_whatsnew_95_de"
```

Comment:
The existing bundle for `de` is needed and should not change as part of this PR. The de page has a different CTA, which needs `mozilla-fxa-product-button`

Reply:
Oops, this was done on accident. Thanks for catching this!